src/loaders/tiff/pixel-source.ts

```diff
@@ -54,19 +54,9 @@ class TiffPixelSource<S extends string[]> implements PixelSource<S> {
      * geotiff.js returns objects with different structure
      * depending on `interleave`. It's weird, but this seems to work.
      */
-    let data = (interleave ? raster : raster[0]) as TypedArray;
-
-    /*
-     * GeoTiff.js returns Uint32Array when the tiff has 32 significant bits,
-     * even if the image is Float32. The underlying ArrayBuffer is correct, but
-     * we need to take a different TypeArray view of the buffer.
-     */
-    if (this.dtype === 'Float32') {
-      data = new Float32Array(data.buffer);
-    }
```

Comment:
What happened here?

Reply:
Mentioned in the PR description but this from a misidentified bug (My bad!!).  For example https://assets.hubmapconsortium.org/8944e654613a8a31bc90d936dd0fade6/ometiff-pyramids/ometiff/20191203_HuBMAP_Spleen/20191203_HuBMAP_Spleen_s0_p14_r1_a1_ac.ome.tif?token= or https://assets.hubmapconsortium.org/1c2b5297ef5c87e0b46cada2c9af62ae/ometiff-pyramids/ometiffs/VAN0011-RK-3-10-IMS_PosMode_multilayer.ome.tif?token= should work fine even though they are 32 bit float.

Reply:
Great!